Lib/test/test_generators.py

```diff
@@ -28,7 +28,8 @@ def generator1(self):
 
     def generator2(self):
         try:
-            yield
+            while True:
+                yield
```

Comment:
I added that when I tried to remove interrupt handling from all resumes. Since I couldn't get all the tests pass that way I reverted that attempt. I'll revert this as well.